AzurePullRequests.Shared/appsettings.json

```diff
@@ -2,9 +2,9 @@
   "ProjectName": "eShopOnWeb",
   "Repository": "", //Guid for RepositoryId
   "OrganizationName": "raphaelmurimigwa",
-  "RestoreStateFile": "2024-04-14-18-21-27.json",
+  "RestoreStateFile": "2024-04-24-18-21-27.json",
```

Comment:
Why did we change the date here?

Reply:
Because it's a 
![images](https://github.com/user-attachments/assets/1a8c0638-632a-4ccf-a824-2baf47e08b4d)